turbo.json

```diff
@@ -2,7 +2,6 @@
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "build": {
-      "dependsOn": ["^build"],
```

Comment:
this seems sus -- if you wanted to build `@glimmer/runtime`, you'd want `@glimmer/owner` to be built first, yeah?